extensions/wikia/Answers/Answers.php

```diff
@@ -638,16 +617,3 @@ function wfAnswersGetEditPointsAjax() {
 	$response->setCacheDuration($wgSquidMaxage);
 	return $response;
 }
-
-$wgHooks["UserToggles"][] = "wfHideFromAttribution";
-function wfHideFromAttribution(&$toggles) {
-	$toggles[] = "hidefromattribution";
-	return true;
-}
-
-
-$wgHooks['CustomArticleFooter'][] = "wfAnswersHideFooter";
```

Comment:
I could not find any invocations of hooks `CustomArticleFooter` and `UserToggles` in our codebase.